packages/cfa-template/template/backend/test/routers/test-auth.test.js

```diff
@@ -83,6 +83,16 @@ describe('auth router -- test-auth', () => {
             title: 'Authentication Error',
         });
     });
+
+    it('reports status OK if it returns options', async () => {
```

Comment:
Right now this test is more saying "it was a successful request", not asserting that actual options were returned.

That would require inspecting the body to confirm that there is an object returned.

That one is probably non-obvious... you'll need to make the mockIntegrationManager respond to getConfigOptions with a body that looks like:
```
{
    "jsonSchema": { ... // some json schema },
    "uiSchema": { ... // optional, but some valid uiSchema }
}
```
It's not obvious since it's not documented anywhere that that's our go-to default (jsonSchema and uiSchema)

Also, bit of a challenge to figure out where to mock out that getConfigOptions request, but figured you might be able to tease that out given the other functions